Mathlib/Data/Real/EReal.lean

```diff
@@ -909,30 +956,37 @@ theorem neg_strictAnti : StrictAnti (- · : EReal → EReal) :=
 protected theorem neg_le {a b : EReal} : -a ≤ b ↔ -b ≤ a := by
  rw [← neg_le_neg_iff, neg_neg]
 
-/-- if `-a ≤ b` then `-b ≤ a` on `EReal`. -/
+/-- `-a ≤ b → -b ≤ a` on `EReal`. -/
 protected theorem neg_le_of_neg_le {a b : EReal} (h : -a ≤ b) : -b ≤ a := EReal.neg_le.mp h
 
-/-- `a ≤ -b → b ≤ -a` on ereal -/
-theorem le_neg_of_le_neg {a b : EReal} (h : a ≤ -b) : b ≤ -a := by
-  rwa [← neg_neg b, EReal.neg_le, neg_neg]
+/-- `a ≤ -b ↔ b ≤ -a` on `EReal`. -/
+protected theorem le_neg {a b : EReal} : a ≤ -b ↔ b ≤ -a := by
+  rw [← neg_le_neg_iff, neg_neg]
 
-/-- Negation as an order reversing isomorphism on `EReal`. -/
-def negOrderIso : EReal ≃o ERealᵒᵈ :=
-  { Equiv.neg EReal with
-    toFun := fun x => OrderDual.toDual (-x)
-    invFun := fun x => -OrderDual.ofDual x
-    map_rel_iff' := neg_le_neg_iff }
+/-- `a ≤ -b → b ≤ -a` on `EReal`. -/
+protected theorem le_neg_of_le_neg {a b : EReal} (h : a ≤ -b) : b ≤ -a := EReal.le_neg.mp h
 
+/-- `-a < b ↔ -b < a` on `EReal`. -/
 theorem neg_lt_comm {a b : EReal} : -a < b ↔ -b < a := by rw [← neg_lt_neg_iff, neg_neg]
 
 @[deprecated (since := "2024-11-19")] alias neg_lt_iff_neg_lt := neg_lt_comm
 
-theorem neg_lt_of_neg_lt {a b : EReal} (h : -a < b) : -b < a := neg_lt_comm.1 h
+/-- `-a < b → -b < a` on `EReal`. -/
+protected theorem neg_lt_of_neg_lt {a b : EReal} (h : -a < b) : -b < a := neg_lt_comm.mp h
 
```

Comment:
why the `_comm`?

Reply:
I originally renamed the theorem to `neg_lt` in this PR to be consistent with `neg_le` and the same lemmas for additive groups, but a few days ago the new name `neg_lt_comm` was approved (#18879) and I did not want to revert the decision.
However `neg_lt` and `lt_neg` look like most natural names for these results, so maybe we should just change them.
What do you think? 
And if we change them, how should we handle the double deprecation on the first one?